pkg/authorization/bootstrap/policy.go

```diff
@@ -39,6 +39,9 @@ const (
 	// This group allows it to skip the entire authorization stack except the bootstrap policy authorizer.
 	// Otherwise, the requests would be rejected because the LogicalCluster resource does not exist yet.
 	SystemLogicalClusterAdmin = "system:kcp:logical-cluster-admin"
+	// SystemExternalLogicalClusterAdmin is a group used by the scheduler to manage LogicalCluster resources.
```

Comment:
can we be a bit more precise what scheduler we mean?

Reply:
actually, we lack the same precision in the paragraph above as well :D

Reply:
additionally it would be great if we could expand on the difference to `system:kcp:logical-cluster-admin`.